app/services/reports/adult_incontinence_report_service.rb

```diff
@@ -131,16 +131,17 @@ def total_people_served_with_loose_supplies_per_month
         .for_year(year)
         .joins(line_items: :item)
         .merge(Item.adult_incontinence)
-        .sum('line_items.quantity / COALESCE(items.distribution_quantity, 50)') / 12
+        .sum('line_items.quantity / COALESCE(items.distribution_quantity, 50)')
     end
 
     def total_people_served_with_supplies_from_kits_per_month
       organization
-      .distributions.for_year(year)
-      .joins(line_items: {item: :kit})
-      .merge(Item.adult_incontinence)
-      .where.not(items: {kit_id: nil})
-      .sum('line_items.quantity / COALESCE(items.distribution_quantity, 1)') / 12
+        .distributions
+        .for_year(year)
+        .joins(line_items: {item: :kit})
+        .merge(Item.adult_incontinence)
+        .where.not(items: {kit_id: nil})
+        .sum('line_items.quantity / COALESCE(items.distribution_quantity, 1)')
```

Comment:
I  *think* this is giving you the people served with items that are kits and adult incontinence both?  (which would be the zero we are getting).      The distinction between items that are kits and items that are in the kits is our bugbear here.

Reply:
There is a problem in the initial count — with what you’re excluding.    But when you have that done,   I’m wondering if using the same sql query, but substituting  
SELECT SUM(COALESCE(line_items.quantity / items.distribution_quantity, 1))  
for 
SELECT SUM(line_items.quantity * kit_line_items.quantity)
 in that same SQL query would do the job.     I am in no way 100% sure on this - but leave it to you to do a reasoning through.      I think line items there is the number of kits in the distributions, and items is the item that represents the kit, yes?